fake-manage-recalls-api/stubs/__files/get-recall.json

```diff
@@ -120,5 +120,8 @@
   "dossierCreatedByUserId": "00000000-2222-0000-0000-000000000000",
   "recallAssessmentDueDateTime": "2020-08-06T15:33:57.000Z",
   "dossierTargetDate": "2020-12-14",
-  "status": "RECALL_NOTIFICATION_ISSUED"
+  "status": "RECALL_NOTIFICATION_ISSUED",
```

Comment:
can you re-run `npm run pact-test` to regenerate the contract?

Reply:
sure